docs/Contributors_Guide/github_workflow.rst

```diff
@@ -234,37 +234,39 @@ Open a pull request using a browser
 
     * An authorized developer will accept the pull request (if
       everything meets acceptance criteria) and merge your code into the remote
-      repository's develop branch.
+      repository's 'develop' branch.
 
 Approve a pull request using a browser
 --------------------------------------
 
 Submitting a pull request allows a user to propose changes, request a
-review of the contributions and have the proposed changes merged into a
+review of the contributions, and have the proposed changes merged into a
 different branch. Pull requests allow the reviewer to see the differences
 in the content from both branches.
 
-For issues with sub-tasks, it may be desired to get some changes into the
-develop branch sooner, rather than later. If the changes seem to make sense
+For issues with sub-tasks, it may be desirable to get some changes into the
+'develop' branch sooner, rather than later. If the changes seem to make sense
 on their own and don't rely on other sub-tasks to be complete, creating a pull
 request for a sub-task may make sense. If a sub-task does rely on other
 sub-tasks to be complete, then it may be best to wait to create a pull request
 until the other sub-tasks are also complete.
 
 
-Creating a pull request
-^^^^^^^^^^^^^^^^^^^^^^^
+Approving a pull request
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
 1.  Click on the “Pull requests” tab in the GitHub repository and
     click on the assigned pull request.
 2.  Ensure the continuous integration (CI) tests from
-    `GitHub Actions <https://github.com/ufs-community/workflow-tools/actions>`_ have
-    passed.  See "All checks have passed" in the figure below. If the tests
+    `GitHub Actions <https://github.com/ufs-community/workflow-tools/actions>`__ have
+    passed. See "All checks have passed" in the figure below. If the tests
     were not successful or if there are conflicts with the base branch,
-    ask the requestor make changes.
+    ask the requestor to make changes.
+
+    .. COMMENT: There is no figure here despite what #2 says. 
```

Comment:
This may have been an issue in porting over the Contributor's guide from the MetPlus version. Here is the link to those docs that have a figure. 
https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html#creating-a-pull-request


Completely an aside...I am not sure if we reference how much of this we borrowed from MetPlus. @jprestop Do you remember crediting MetPlus with this content?

Reply:
@christinaholtNOAA There was an acknowledgements section with nothing it it (which I deleted). Should I add it back with a thank you to METplus?

Reply:
Added the figure and removed the comment!

Reply:
@gspetro-NOAA I think an acknowledgement would be a fair way to handle the MetPlus reference. Thank you!